scripts/verify-binary-compatibility

```diff
@@ -6,13 +6,13 @@ self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
 source "$self_dir"/utils.sh
 
 # Note: this should be kept in sync with build.gradle -> patchPluginXml.
-# We're not including any 201.* (aka 2020.1) release here since that's the version we're currently building against anyway
+# We're not including release that's the version we're currently building against anyway
```

Comment:
```suggestion
# We're not including the release that we're currently building against (see build.gradle -> allprojects.ext.intellijVersion) 
since for this case binary compatibility is essentially guaranteed by the compiler
```